src/Microsoft.Restier.Core/Conventions/ConventionBasedApiModelBuilder.cs

```diff
@@ -137,8 +136,7 @@ private IQueryable GetSingletonQuery(QueryExpressionContext context)
                 object target = null;
                 if (!singletonProperty.GetMethod.IsStatic)
                 {
-                    target = context.QueryContext.ApiContext
-                        .GetProperty(typeof(Api).AssemblyQualifiedName);
+                    target = context.QueryContext.GetApiService<IApi>();
```

Comment:
Just for design discussion: Since we already have an abstract class called `ApiBase` which is used as the base class for all the `Api` classes (we don't expect the user to directly implement an `Api` class from `IApi`), do you think we should still keep `IApi`? I didn't see its value here.